### DISCUSSION TITLE
How to build and install jotai locally for testing changes?

### DISCUSSION THREAD

**Discussion by reuben** [pmndrs/jotai#1533] [Q&A]
I've tried following [CONTRIBUTING.md](https://github.com/pmndrs/jotai/blob/main/CONTRIBUTING.md) and using `yarn link` or `yarc`, but both lead to errors when running webpack on my project consuming the local version:

```
  ModuleNotFoundError: Module not found: Error: Can't resolve 'jotai' in '...'
```

Things I've tried:

- `yarn install && yarn build && yarn pack` in jotai, then reference package in my package.json with `file:./deps/jotai-v1.9.1.tgz`
- `yarn install && yarn build && gtar cvzf jotai-v1.9.1.tgz --transform 's,^dist,package,' dist`, then reference as above
- `yarn link` and then `yarn link jotai` in my project
- `yalc publish` and then `yalc add jotai` in my project

All cause the same error.

**Comment by dai-shi** [comment]
@TheEhsanSarshar Can you help?

**Comment by byteab** [comment]
@reuben 
could you please share a demo?

**Reply by reuben** [reply]
Not easily. I workaround by editing node_modules directly.

**Reply by byteab** [reply]
take a look into this
https://stackoverflow.com/a/72343913/10539792
this might be the issue
